cfgov/unprocessed/css/molecules/global-search.less

```diff
@@ -0,0 +1,226 @@
+                background: @gray-10;
+                border: 1px solid @gray-50;
+                border-bottom: none;
+                color: @black;
+
+                .cf-icon:before {
+                    content: @cf-icon-delete;
+                }
+
+                // Cover up the border edge of the flyout so that it is attached
+                // visually to the trigger button.
+                &:after {
+                    position: absolute;
+                    left: 0;
+                    bottom: -1px;
+                    content: '\a0';
+                    display: block;
+                    height: 1px;
+                    width: 100%;
+                    background: @gray-10;
+                }
+            }
+        } );
+
+        // Show "Close" text in trigger at tablet sizes.
+        .respond-to-range( @bp-sm-min, @bp-med-min, {
+            &[aria-expanded="true"] {
+                .m-global-search_trigger-label:before {
+                    content: 'Close';
+                }
+            }
+        } );
+
+        .respond-to-min( @bp-med-min {
+            &[aria-expanded="true"] {
+                display: none;
+            }
+        } );
+    }
+
+    &_content {
+        display: none;
+
+        &[aria-expanded="true"] {
+            display: block;
+        }
+
+        // Create flyout at table/mobile size.
+        .respond-to-max( @bp-sm-max, {
+            width: 100%;
+            box-sizing: border-box;
+
+            position: absolute;
+            left: 0;
+            // Offset for height of trigger button.
+            top: 60px;
+
+            background-color: @gray-10;
+            border: 1px solid @gray-50;
+            pointer-events: auto;
+            transform: translateX(100%);
+            transition: transform 0.2s ease-in,
+                        visibility 0s 0.5s;
+            visibility: hidden;
+            z-index: 10;
+
+            &-form {
+                padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
+                padding-left: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+                padding-right: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+                padding-bottom: unit( @grid_gutter-width / 2 / @base-font-size-px, em );
+
+                // TODO: Move these styles to cf-enhancements
+                .input-with-btn_btn {
+                    border-left: 0;
+                    button {
+                        border-top-left-radius: 0;
+                        border-bottom-left-radius: 0;
+                    }
+                }
+            }
+
+            // Drop-shadow under the search flyout menu at mobile/tablet size.
+            &:after {
+                position: relative;
+                top: 5px;
+                content: '\a0';
+                display: block;
+                height: 5px;
+                width: 100%;
+                background: rgba( 90, 93, 97, 0.2 );
```

Comment:
Updated to use `@gray` and `opacity` rule in https://github.com/cfpb/cfgov-refresh/commit/6a126448e3207a204c075bc77fb1948ac2943b83